Adding egress definition (#442)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

// Fuse: Prefixed second sentence with "In Red Hat Fuse, an EIP is a" (Breda)
// Fuse: Changed "Camel" to "Red Hat Fuse" (Breda)
// Fuse: Removed the final sentence about the Palette (Breda)

[[egress]]

> ==== image:images/yes.png[yes] egress (adjective, noun)

Reviewer: From looking at other entries, the "noun" and "adjective" defintions are separate. WOuld you be OK to follow the same pattern?

> *Description*: In Red Hat OpenShift, _egress_ describes the process of network traffic leaving an OpenShift cluster. Alternatively, data _ingress_, refers to outside network traffic entering an OpenShift cluster.

Reviewer: ```suggestion
*Description*: In Red{nbsp}Hat OpenShift, _egress_ describes the process of network traffic leaving an OpenShift cluster. Alternatively, data _ingress_, refers to outside network traffic entering an OpenShift cluster.
```

> *Description*: In Red Hat OpenShift, _egress_ describes the process of network traffic leaving an OpenShift cluster. Alternatively, data _ingress_, refers to outside network traffic entering an OpenShift cluster.

Author: Done :)

> *Description*: In RedHat OpenShift, _egress_ describes the outgoing direction of network traffic. In Red Hat OpenShift, for example, this traffic is described as entering (ingress) or leaving (egress) an OpenShift cluster.

Reviewer: ```suggestion
*Description*: In RedHat OpenShift, _egress_ describes the outgoing direction of network traffic. For example, traffic is described as entering (ingress) or leaving (egress) an OpenShift cluster.
```

Author: I like Brian's suggestion as well, so i'm going to combine these two. Thank you!

Author: Also I saw your comment about parts of speech, and I agree I think I can remove the noun bc I removed the references to specific OpenShift product area names. This definition is really just an adjective now. For the Ingress definition, I think we settled on using (noun, adjective) instead of creating a separate entry. But I think there's not many others that do that.

> *Description*: In RedHat OpenShift, _egress_ describes the outgoing direction of network traffic. In Red Hat OpenShift, for example, this traffic is described as entering (ingress) or leaving (egress) an OpenShift cluster.

Reviewer 2: ```suggestion
*Description*: In Red{nbsp}Hat OpenShift, _egress_ describes the process of network traffic leaving an OpenShift cluster. Data _ingress_, on the other hand, refers to outside network traffic that enters an OpenShift cluster.
```
- Consider this revision as a way of simplifying the contrast between _egress_ and _ingress_.
- Add non-breaking space code between Red and Hat.

Reviewer: Thanks, Brian. Please swap "other hand" with one of the following IBM SG suggestions:

![Uploading Screenshot from 2024-03-20 09-24-02.png…]()

// Fuse: Prefixed second sentence with "In Red Hat Fuse, an EIP is a" (Breda)
// Fuse: Changed "Camel" to "Red Hat Fuse" (Breda)
// Fuse: Removed the final sentence about the Palette (Breda)

[[egress]]
==== image:images/yes.png[yes] egress (adjective)

> *Description*: In Red{nbsp}Hat OpenShift, _egress_ describes the process of network traffic leaving a Red{nbsp}Hat OpenShift cluster. Alternatively, data _ingress_, refers to outside network traffic entering an OpenShift cluster.

Reviewer 2: ```suggestion
*Description*: In Red{nbsp}Hat OpenShift, _egress_ describes the process of network traffic leaving a Red{nbsp}Hat OpenShift cluster. Alternatively, data _ingress_ refers to outside network traffic entering an OpenShift cluster.
```
- Remove comma between subject and verb

Author: Nice catch, thanks! Addressed.
